client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab/index.ts

```diff
@@ -156,6 +157,50 @@ const usePersonalInfoTab = (): usePersonalInfoTabOutcome => {
 
     const clearSubOccupations = () => setSubOccupations([]);
 
+    const updateInvestigationReasonId = (epidemiologyNumber: number, newComplexityReasonId: number) => {
+        axios.post('/investigationInfo/updateComplexityReason', {
+            epidemiologyNumberInput: epidemiologyNumber,
+            newComplexityReasonId: newComplexityReasonId})
+            .then(() => {})
+            .catch((err) => {console.log(err)})
+    }
+
+    const removeInvestigationReasonId = (epidemiologyNumber: number, oldComplexityReasonId: number) => {
+        axios.post('/investigationInfo/deleteComplexityReason', {
+            epidemiologyNumberInput: epidemiologyNumber,
+            oldComplexityReasonId: oldComplexityReasonId})
+            .then(() => {})
+            .catch((err) => {console.log(err)})
+    }
```

Comment:
add a logger and maybe move those functions to a hook? those seem similar to the functions in useExposuresSaving